source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -157,5 +157,72 @@ the Query API:
                                       then: "extreme temperature",
                                       else: "no advisory" } } } }
 
+String Operations
+~~~~~~~~~~~~~~~~~
+
+You can perform a string operation on a value of type ``MqlString``
+using the methods described in this section.
+
+The following table describes string methods
+available in the driver and corresponding expressions in the
+Query API. The methods link to API documentation and the Query API
+expressions link to Server manual documentation.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Method
+     - Aggregation Pipeline Operator
+
+   * - `append() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#append(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$concat </reference/operator/aggregation/concat/>`
+       
+   * - `toLower() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#toLower()>`__
+     - :manual:`$toLower </reference/operator/aggregation/toLower/>`
+
+   * - `toUpper() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#toUpper()>`__
+     - :manual:`$toUpper </reference/operator/aggregation/toUpper/>`
+
+   * - `length() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#length()>`__
+     - :manual:`$strLenCP </reference/operator/aggregation/strLenCP/>`
+
+   * - `lengthBytes() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#lengthBytes()>`__
+     - :manual:`$strLenBytes </reference/operator/aggregation/strLenBytes/>`
+
+   * - `substr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#substr(int,int)>`__
+     - :manual:`$substrCP </reference/operator/aggregation/substrCP/>`
+
+   * - `substrBytes() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#substrBytes(int,int)>`__
+     - :manual:`$substrBytes </reference/operator/aggregation/substrBytes/>`
+
+.. include:: /includes/fundamentals/agg-mql-diff.rst
+   
+The following example creates a unique employee username from their last
+name and unique employee ID. The code uses the ``append()`` method to
```

Comment:
Suggestion:

I think this can be made shorter as something unique appended with anything else should also be unique (except in special cases which this operation doesn't handle). I think it's ok to even leave out "unique" since that doesn't seem central to the append operation nor the understanding of it. E.g.

"This example generates a lowercase username from the employee's last name and their employee ID."

Then, the rest of the detailed information about what each operation does could be moved to a separate paragraph to provide a detailed breakdown.